src/Package/Impl/Rules/Run.cs

```diff
@@ -0,0 +1,270 @@
+//------------------------------------------------------------------------------
+// <auto-generated>
+//     This code was generated by a tool.
+//     Runtime Version:4.0.30319.42000
+//
+//     Changes to this file may cause incorrect behavior and will be lost if
+//     the code is regenerated.
+// </auto-generated>
+//------------------------------------------------------------------------------
+
+// WARNING: this file was auto-generated then modified.
```

Comment:
I sent out an email to cps mailing list regarding my troubles with XamlPropertyRule build action.  I haven't heard back yet.